src/components/home/homepage.tsx

```diff
@@ -0,0 +1,9 @@
+const HomePage = () => {
+  return (
+    <div className="mx-auto flex h-screen w-screen items-center justify-center text-center text-[4vw] text-black">
```

Comment:
```suggestion
    <div className="mx-auto flex w-screen items-center justify-center text-center text-[4vw] text-black">
```

style the text a bit more so the distance to the main text and size relative to the main text match the figma; looks good otherwise